<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EditGameDialog</class>
 <widget class="QDialog" name="EditGameDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>350</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Editace</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <widget class="QLabel" name="label_8">
         <property name="font">
          <font>
           <pointsize>16</pointsize>
           <bold>true</bold>
          </font>
         </property>
         <property name="frameShape">
          <enum>QFrame::Shape::NoFrame</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Shadow::Plain</enum>
         </property>
         <property name="lineWidth">
          <number>2</number>
         </property>
         <property name="midLineWidth">
          <number>0</number>
         </property>
         <property name="text">
          <string>Editování hry</string>
         </property>
         <property name="textFormat">
          <enum>Qt::TextFormat::AutoText</enum>
         </property>
         <property name="alignment">
          <set>Qt::AlignmentFlag::AlignCenter</set>
         </property>
         <property name="margin">
          <number>3</number>
         </property>
         <property name="indent">
          <number>0</number>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_5">
           <item>
            <spacer name="verticalSpacer_4">
             <property name="orientation">
              <enum>Qt::Orientation::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>40</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QLabel" name="label_9">
             <property name="text">
              <string>Název</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="editNameLineEdit"/>
           </item>
           <item>
            <spacer name="verticalSpacer_5">
             <property name="orientation">
              <enum>Qt::Orientation::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>40</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QLabel" name="label_10">
             <property name="text">
              <string>Studio</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="editStudioLineEdit"/>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_6">
           <item>
            <widget class="QLabel" name="label_11">
             <property name="text">
              <string>Žánr</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="editGenreComboBox"/>
           </item>
           <item>
            <widget class="QLabel" name="label_12">
             <property name="text">
              <string>Rok vydání</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QSpinBox" name="editYearSpinBox">
             <property name="minimum">
              <number>1900</number>
             </property>
             <property name="maximum">
              <number>2030</number>
             </property>
             <property name="value">
              <number>2025</number>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_13">
             <property name="text">
              <string>Hodnocení</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QSpinBox" name="editRatingSpinBox">
             <property name="maximum">
              <number>5</number>
             </property>
             <property name="value">
              <number>3</number>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="verticalSpacer_6">
         <property name="orientation">
          <enum>Qt::Orientation::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QLabel" name="label_14">
         <property name="text">
          <string>Obrázek / ikonka hry (nepovinné)</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="editImageLineEdit">
         <property name="text">
          <string>Cesta k obrázku</string>
         </property>
         <property name="echoMode">
          <enum>QLineEdit::EchoMode::Normal</enum>
         </property>
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="editBrowseButton">
         <property name="text">
          <string>Procházet...</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QDialogButtonBox" name="editButtonBox">
         <property name="standardButtons">
          <set>QDialogButtonBox::StandardButton::Cancel|QDialogButtonBox::StandardButton::Ok</set>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
